src/services/dumpers/agent-nodejs.ts

```diff
@@ -108,19 +115,23 @@ export default class AgentNodeJs extends AbstractDumper {
       isMSSQL: dbDialect === 'mssql',
       isMariaDB: dbDialect === 'mariadb',
       forestServerUrl: this.env.FOREST_URL_IS_DEFAULT ? false : this.env.FOREST_SERVER_URL,
-      datasourceImport: isMongoose
-        ? `const { createMongooseDataSource } = require('@forestadmin/datasource-mongoose');\nconst connection = require('./mongoose-models');`
-        : `const { createSqlDataSource } = require('@forestadmin/datasource-sql');`,
-      datasourceCreation: isMongoose
-        ? 'createMongooseDataSource(connection, {})'
-        : `
+      datasourceImport: null,
+      datasourceCreation: null,
+    };
+
+    if (isMongoose) {
+      context.datasourceImport = `const { createMongooseDataSource } = require('@forestadmin/datasource-mongoose');\nconst primaryConnection = require('./models/primary');`;
+      context.datasourceCreation = `createMongooseDataSource(primaryConnection, { flattenMode: 'auto' })`;
```

Comment:
So, models will be written in `./models/primary`? Any reason for the `primary`naming ?

Reply:
Nop. I've just added the structure to add multi-database support by default. Primary is basically a name, could be `default`, `first-one`, even your name if you want.